packages/vscode-extension/src/debugging/DebugAdapter.ts

```diff
@@ -413,6 +547,40 @@ export class DebugAdapter extends DebugSession {
         return this.cdpSession.sendCDPMessage("Runtime.getProperties", params);
       }
     );
+
+    if (args.filter === "indexed") {
```

Comment:
Second variable makes the call to CDP to execute method that returns part of the array, based on: https://github.com/microsoft/vscode-js-debug/blob/6902aae25b4970d137041f9a84221eb327dec5a9/src/adapter/variableStore.ts#L1033
and remote call: 
https://github.com/microsoft/vscode-js-debug/blob/8b0bffc47d1836f4357db9d923685e730630a57a/src/adapter/templates/index.ts#L127

Reply:
don't put explanation in comments. If something needs to be explain, it should either be in code as inline comment, or in the PR description that gets included in the git history. Inline comments on PRs like this one are not something people typically look at as you typically check the code and then the blame commit + the comments are automatically folded when you click "resolve"